tests/system_tests_three_routers.py

```diff
@@ -215,6 +218,15 @@ def test_04_linkroute ( self ):
         test.run()
         self.assertEqual(None, test.error)
 
+    def test_05_linkroute ( self ):
```

Comment:
The name of this test should not be test_05_linkroute because it is using message routing